### PR TITLE
DELETE THIS PR I have split the two issues 

### LINKED ISSUE #87: Could not find group puppet
Hi,

It seems that the puppet user/group is not available anymore since the migration to the puppet-agent package:

```
Error: Could not find group puppet
Error: /Stage[main]/Puppet/File[/opt/puppetlabs/facter]/group: change from root to puppet failed: Could not find group puppet
Notice: /Stage[main]/Puppet/File[/opt/puppetlabs/facter/facts.d]: Dependency File[/opt/puppetlabs/facter] has failures: true
Warning: /Stage[main]/Puppet/File[/opt/puppetlabs/facter/facts.d]: Skipping because of failed dependencies
```

```
 # rpm -qa puppet-agent
puppet-agent-1.4.2-1.el6.x86_64
```

The problem occurs on RedHat and Debian based OS.

Best,
Jerome

### PR DESCRIPTION
**Pull request by chrisowensboston**
OK to delete this PR I have separated the two fixes into separate PRs
Made profile/puppetdb align with profile/master by adding param puppetdb_manage_dbserver (default true) that is passed to puppetdb.  Also fixes #87

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -215,11 +215,18 @@
     include ::puppet::repo
     Class['::puppet::repo'] -> Class['::puppet::install']
   }
+
+  if $allinone {
+    $owner_group = 'root'
+  }
+  else {
+    $owner_group = 'puppet'
+  }
```
Comment (rendhalver) [inline manifests/init.pp]: I would prefer this in a separate PR.

File: Gemfile.lock [hunk 2]
```diff
@@ -151,6 +151,7 @@ DEPENDENCIES
   facter
   hiera
   json (= 1.8.3)
+  json_pure (< 2.0.2)
```
Comment (rendhalver) [inline Gemfile.lock]: Is there a Gemfile change to go with this?

**Comment by chrisowensboston** [comment]
OK. Look for the separate PR tomorrow.

**Comment by chrisowensboston** [comment]
I don’t know why Gemfile.lock changed. I did not change Gemfile; I just ran bundle update.

**Pull request closed without merging**